src/bugbug/data/default.nix

```diff
@@ -48,7 +48,6 @@ let
         ];
         deadline = "24 hours";
         maxRunTime = 24 * 60 * 60;
-        workerType = "releng-svc-compute";
```

Comment:
You don't use this specific worker type anymore ?

Reply:
I found it useless now, as I've reduced the number of commits that I mine from mercurial (I used to mine since commit 0, now only the last 2 years and a half).
So at the moment the runtime of the task is dominated by the download of bugs from Bugzilla.

I'm using releng-svc-compute for the training task.